src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -76,7 +77,7 @@ protected void onEvent(GHEvent event, String payload) {
         return new Function<GHEventsSubscriber, Set<GHEvent>>() {
             @Override
             public Set<GHEvent> apply(GHEventsSubscriber subscriber) {
-                return defaultIfNull(subscriber.events(), Collections.<GHEvent>emptySet());
+                return defaultIfNull(notNull(subscriber).events(), Collections.<GHEvent>emptySet());
```

Comment:
can you then use the `notNull(Object, String)` so we may have some clue as to what is null in the stack trace - otherwise just supress is actually more explicit that you intent this to barf and not be fixed.